fairlearn/reductions/_grid_search/grid_search.py

```diff
@@ -126,10 +127,14 @@ def fit(self, X, y, **kwargs):
             logger.debug("Regression problem detected")
             is_classification_reduction = False
 
-        _, y_train, sensitive_features_train = _validate_and_reformat_input(
-            X, y, enforce_binary_labels=is_classification_reduction, **kwargs)
+        _, y_train, sensitive_features_train, control_features_train = \
+            _validate_and_reformat_input(X, y,
+                                         enforce_binary_labels=is_classification_reduction,
+                                         **kwargs)
 
         kwargs[_KW_SENSITIVE_FEATURES] = sensitive_features_train
+        if control_features_train is not None:
+            kwargs[_KW_CONTROL_FEATURES] = control_features_train
```

Comment:
After `_validate_and_reformat_input` is moved into `Moment.load_data`, we should be able to get rid of this chunk of the code.

Reply:
Done